content/kb/how-to-migrate-longhorn-chart-installed-in-old-rancher-ui-to-the-chart-in-new-rancher-ui.md

```diff
@@ -0,0 +1,206 @@
+---
+title: "Instruction: How To Migrate Longhorn Chart Installed In Old Rancher UI To The Chart In New Rancher UI"
+author: Phan Le
+draft: false
+date: 2022-06-23
+categories:
+- "instruction"
+- "upgrade"
+- "migration"
+---
+
+## Applicable versions
+
+Longhorn v1.2.x and v1.3.x versions
+
+## Scenario
+In the past, you had a Rancher v2.5.x instance running.
+This Rancher instance was managing a few downstream clusters.
+In one of the downstream clusters, you installed Longhorn using the old Rancher UI (the App Catalog UI) as shown in the below picture:
+
+{{< figure src="/img/kb/how-to-migrate-longhorn-chart-installed-in-old-rancher-ui-to-the-chart-in-new-rancher-ui/longhorn-chart-in-old-rancher-ui.png" >}}
+
+Now, you have already upgraded Rancher to the version >= v2.6.3.
+In this new Rancher version, a new UI, App & Marketplace, was introduced to replace the old App Catalog UI.
+Rancher hasn't removed the App Catalog UI completely yet.
+You can still find your old Longhorn installation by going to Downstream Cluster ->
+Legacy -> Project -> select the correct project name from the dropdown on the top -> Apps. For example:
+
+{{< figure src="/img/kb/how-to-migrate-longhorn-chart-installed-in-old-rancher-ui-to-the-chart-in-new-rancher-ui/longhorn-installation-in-legacy-ui.png" >}}
+
+At this moment, there are 2 different Longhorn charts. One is in the App Catalog UI (you already installed Longhorn using this chart). The other one in the App & Marketplace (you have not installed Longhorn using this chart) as below:
+
+{{< figure src="/img/kb/how-to-migrate-longhorn-chart-installed-in-old-rancher-ui-to-the-chart-in-new-rancher-ui/longhorn-chart-in-new-ui.png" >}}
+
+Because the App Catalog UI is being deprecated, you may want to migrate the existing Longhorn installation to the Longhorn chart in the App & Marketplace UI.
+The reason is Longhorn is planing to stop releasing new versions in the App Catalog UI in the future.
+
+
+## Prerequisites
+1. Longhorn v1.2.x and v1.3.x versions
+2. Rancher >= v2.6.3 versions
+3. The command [kubectl get-all](https://github.com/corneliusweig/ketall#installation) is available on your local machine
+
+## Steps
+
+1. Go to Longhorn UI -> Settings -> General -> Change Concurrent Automatic Engine Upgrade Per Node Limit setting to 0 to disable auto engine upgrade.
```

Comment:
NIT: Indicate how to use `kubectl` to update this.

Reply:
Thank you. Fixed